.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 
 os:
-  - osx
```

Comment:
It was in the default, removed it because a build will take twice as long. I do not expect different behavior from node on osx/linux and since it will also not run in windows anyway I thought just linux would be enough. You can add it if you want to run on osx as well.